### PR TITLE
Fix mushorg/snare/issues/14

### PR DESCRIPTION
**Pull request by rnehra01**
Session tracking using cookies(https://github.com/mushorg/snare/issues/14)

### CODE REVIEW

**Comment by afeena** [comment]
Could you please fix tests?

**Comment by rnehra01** [comment]
@afeena It would be nice if could provide some help about `https://travis-ci.org/mushorg/tanner/builds/210665189#L220`. I've been trying but can't understand.

**Comment by afeena** [comment]
@rnehra01  In `test_server.py`, `session` is mock. Tests try to call function `get_uuid` of mock object. So, you should mock this function too https://github.com/mushorg/tanner/blob/master/tanner/tests/test_server.py#L38

**Comment by rnehra01** [comment]
@afeena done

File: tanner/tests/fc805b0ef8357336e620020c33c3837f [hunk 1]
```diff
@@ -0,0 +1 @@
+1489461547
```
Comment (afeena) [inline tanner/tests/fc805b0ef8357336e620020c33c3837f]: I think this file should not be in the repo :)

**Comment by rnehra01** [comment]
@afeena done

**Comment by afeena** [comment]
Well done ;)

**Pull request merged**